modules/virt-enabling-persistentreservation-feature-gate.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * * virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-enabling-persistentreservation-feature-gate_{context}"]
+= Enabling the PersistentReservation feature gate
+
+You can enable the SCSI PersistentReservation feature gate and allow a LUN-backed block mode virtual machine (VM) disk to be shared among multiple virtual machines. The PersistentReservation feature gate is disabled by default.
+
+[IMPORTANT]
+====
+If you are a cluster administrator and intend to configure disk sharing by using LUN, you must enable the cluster's feature gate on the Hyperconverged Cluster Operator (HCO) Custom Resource (CR). 
+====
+
+.Prerequisites
+
+* You must have cluster administrator privileges. 
+* `ReadWriteMany` (RWX) volume access mode is required if the VMs that are sharing disks are running on different nodes. If the VMs that are sharing disks are running on the same node, `ReadWriteOnce` (RWO) volume access mode is sufficient.
+* The storage provider must support a Container Storage Interface (CSI) driver that uses the SCSI protocol.
+
+
+
+.Procedure
+
+. Run the following command:
+
++
+[source,terminal,subs="attributes+"]
+----
+$ oc patch hyperconverged kubevirt-hyperconverged -n {CNVNamespace} \
+  --type json -p '[{"op":"replace","path":"/spec/featureGate/persistentReservation", "value":"true"}]'
+----
+
+* Configure `.spec.featureGates.persistentReservation=true` on the HCO CR to enable the feature:
+
++
+[source,yaml]
+====
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  certConfig:
+    ca:
+      duration: 48h0m0s
+      renewBefore: 24h0m0s
+    server:
+      duration: 24h0m0s
+      renewBefore: 12h0m0s
+  featureGates:
+    persistentReservation: true    
+  liveMigrationConfig:
+    allowAutoConverge: false
+    allowPostCopy: false
+    completionTimeoutPerGiB: 800
+    parallelMigrationsPerCluster: 5
+    parallelOutboundMigrationsPerNode: 2
+    progressTimeout: 150
+  uninstallStrategy: BlockUninstallIfWorkloadsExist
+  workloadUpdateStrategy:
+    batchEvictionInterval: 1m0s
+    batchEvictionSize: 10
+    workloadUpdateMethods:
+    - LiveMigrate
+  workloads: {}
+---- 
+====
+
+
```

Comment:
If you have not configured VS Code to remove trailing white space, this would be a good time to do so.

Reply:
Done